utils/build.ps1

```diff
@@ -1474,25 +1507,25 @@ function Build-LLBuild($Arch, [switch]$Test = $false) {
       # Build additional llvm executables needed by tests
       Isolate-EnvVars {
         Invoke-VsDevShell $HostArch
-        Invoke-Program ninja.exe -C "$BinaryCache\0" FileCheck not
+        Invoke-Program ninja.exe -C (Get-HostProjectBinaryCache BuildTools) FileCheck not
       }
 
       $Targets = @("default", "test-llbuild")
       $TestingDefines = @{
-        FILECHECK_EXECUTABLE = "$BinaryCache\0\bin\FileCheck.exe";
+        FILECHECK_EXECUTABLE = (Join-Path -Path (Join-Path -Path (Get-HostProjectBinaryCache BuildTools) -ChildPath "bin") -ChildPath "FileCheck.exe");
```

Comment:
The `Join-Path` seem a little overkill throughout?
```suggestion
        FILECHECK_EXECUTABLE = "$(Get-HostProjectBinaryCache BuildTools)\bin\FileCheck.exe";
```

Reply:
Would be nice to be able to support other platforms some day (e.g. Linux)

Reply:
We could create a Join-Path variant that takes an arbitrary number of components and doesn't require labels so it's tidier.

Reply:
or use `[IO.Path]::Combine($foo, $bar, $baz)`

Reply:
Let me try switching to `[IO.Path]::Combine` ... if we can use builtins, I'd prefer that.